crates/runtime/tests/models/snapshots/integration_models__models__search__mulit_column_no_pks_basic_error_response.snap

```diff
@@ -0,0 +1,6 @@
+---
+source: crates/runtime/tests/models/search.rs
+expression: e.to_string()
+snapshot_kind: text
+---
+HTTP error: 500 Internal Server Error - Error occurred aggregating candidate search results: Cannot aggregate candidates from multiple sources (e.g. embedded columns of a dataset) when the dataset has no primary key.
```

Comment:
Is this supposed to be a 500 error?

Reply:
Sure, but I don't think we want a 500 Internal Server Error to be an expected error type. This should be more of a 400 level error.

Reply:
Good point. I can update this.